graphql/README.md

```diff
@@ -40,6 +41,34 @@ Visit the [Developer Guide](https://nasa.github.io/cumulus-orca/docs/developer/d
   - Use the command `docker run -d -p 5000:5000 --env-file path/to/env imageName` 
     - Replace `imageName` with the name of your built image.
     - Replace `path/to/env` with the path to the env file you created in the previous step.
+- The GraphQL URL is `http://0.0.0.0:5000/graphql`
+- 
+### AWS Testing
+1. If you wish to enable the developer GUI, add/modify the "ORCA_ENV" 
+   in the `environment` section in your Terraform's `aws_ecs_task_definition` to
+   ```json
+   {
+     "name": "ORCA_ENV",
+     "value": "development"
+   }
+   ```
+   then redeploy.
+   
+2. Run the following bash command, 
+   replacing `i-00000000000000000` with your ec2 instance name.
```

Comment:
How do we know which instance? Is it the Cumulus deployed EC2 instance ID named as named `<prefix>-Cumulus-ECSCluster`?

Might be worth adding this to get the ID from CLI
  ```bash
  aws ec2 describe-instances --filters Name=instance-state-name,Values=running Name=tag:Name,Values=<PREFIX>-CumulusECSCluster --query "Reservations[*].Instances[*].InstanceId" --output text
  ```

Reply:
Added.